test/colony-funding.js

```diff
@@ -842,28 +833,72 @@ contract("Colony Funding", addresses => {
       await colony.startNextRewardPayout(otherToken.address);
     });
 
+    it("should not be able to finalize the payout if payout is not active", async () => {
+      const tx = await colony.startNextRewardPayout(otherToken.address);
+      const payoutId = tx.logs[0].args.id;
+
+      await forwardTime(5184001, this);
+      await colony.finalizeRewardPayout(payoutId);
+
+      await checkErrorRevert(
+        colony.finalizeRewardPayout(payoutId, {
+          from: userAddress1
+        }),
+        "colony-reward-payout-token-not-active"
+      );
+    });
+
+    it("should not be able to finalize payout if payout is still active", async () => {
+      const tx = await colony.startNextRewardPayout(otherToken.address);
+      const payoutId = tx.logs[0].args.id;
+
+      await checkErrorRevert(colony.finalizeRewardPayout(payoutId), "colony-reward-payout-active");
+    });
+
+    it("should not be able to finalize payout if payoutId does not exist", async () => {
+      await colony.startNextRewardPayout(otherToken.address);
+
+      await checkErrorRevert(colony.finalizeRewardPayout(10), "colony-reward-payout-not-found");
+    });
+
+    it("should return correct info about reward payout", async () => {
+      const { logs } = await colony.startNextRewardPayout(otherToken.address);
+      const payoutId = logs[0].args.id;
+
+      const balance = await colony.getPotBalance(0, otherToken.address);
+      const blockTimestamp = await currentBlockTime();
+
+      const info = await colony.getRewardPayoutInfo(payoutId);
+      assert.equal(info[1].toString(), userTokens.toString());
```

Comment:
Is there a reason we're not checking `info[0]`? We could compare it to `colonyNetwork.getReputationRootHash()`, right? (Although without going through all the mining setup, I suppose that's going to just be `0x0`, which might be the reason you've skipped it?)

Reply:
Yea, i thought i can add it when we add proofs (which will require mining process), but i guess we can do it now